zerver/webhooks/github/tests.py

```diff
@@ -386,14 +386,15 @@ def test_pull_request_assigned_msg_with_custom_topic_in_url(self) -> None:
         self.check_webhook("pull_request__assigned", expected_topic_name, expected_message)
 
     def test_pull_request_unassigned_msg(self) -> None:
-        expected_message = (
-            "eeshangarg unassigned [PR #1](https://github.com/zulip-test-org/helloworld/pull/1)."
-        )
-        self.check_webhook(
-            "pull_request__unassigned",
-            "helloworld / PR #1 Mention that Zulip rocks!",
-            expected_message,
-        )
+        expected_message = "eeshangarg unassigned eeshangarg from [PR #1](https://github.com/zulip-test-org/helloworld/pull/1)."
+        expected_topic_name = "helloworld / PR #1 Mention that Zulip rocks!"
```

Comment:
Only **had** to change "unassigned" to "unassigned <user> from", in this test case.
Other changes are to refactor to use `expected_topic_name`.